docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -252,10 +252,10 @@ to set the index:
 ------------------------------------------------------------------------------
 output.elasticsearch:
   hosts: ["http://localhost:9200"]
-  index: "%\{[fields.log_type]\}-%\{[beat.version]\}-%\{+yyyy.MM.dd}\" <1>
+  index: "%\{[fields.log_type]\}-%{[{beat_version_key}]}-%\{+yyyy.MM.dd}\" <1>
 ------------------------------------------------------------------------------
 
-<1> We recommend including `beat.version` in the name to avoid mapping issues
+<1> We recommend including `{beat_version_key}` in the name to avoid mapping issues
```

Comment:
Are you expecting this attribute to be conditionally rendered? Right now we're showing both `observer.version` and `{beat_version_key}`. As an example:

<img width="774" alt="screen shot 2019-01-28 at 10 55 57 am" src="https://user-images.githubusercontent.com/5618806/51859182-66457e00-22eb-11e9-9a9b-bc6368b176c6.png">

Reply:
oh, I missed that one. Yes I'd expect it to show up as `observer_version` every where. Some background: the beats team is using `agent.version` from 7.0, and we use `observer.version`. My intention is to avoid having yet another `ifndef` block in the beats docs, and make `agent.version` default but overridable.

Reply:
Perfect, just making sure. Looks like there are four instances of incorrect rendering. I've made comments below on how to fix each. This should allow for easy substitution in the beats docs without any new conditionals.